pkg/microservice/aslan/core/workflow/testing/service/openapi.go

```diff
@@ -91,3 +122,66 @@ func generateScanningModuleFromOpenAPIInput(req *OpenAPICreateScanningReq, log *
 	ret.Repos = repoList
 	return ret, nil
 }
+
+func OpenAPICreateTestTask(userName string, args *OpenAPICreateTestTaskReq, logger *zap.SugaredLogger) (int64, error) {
+	task := &commonmodels.TestTaskArgs{
+		TestName:        args.TestName,
+		ProductName:     args.ProjectName,
+		TestTaskCreator: userName,
+	}
+	result, err := CreateTestTask(task, logger)
+	if err != nil {
+		logger.Errorf("OpenAPI: user:%s failed to create project:%s test task:%s, err: %s", userName, args.ProjectName, args.TestName, err)
```

Comment:
the error message should be like:
.... failed to create test, project: %s, test : %s, err: ....    or 
..... test creation failed, project: ......